x-pack/plugins/stack_alerts/public/rule_types/es_query/expression/esql_query_expression.tsx

```diff
@@ -61,12 +58,11 @@ export const EsqlQueryExpression: React.FC<
     groupBy: DEFAULT_VALUES.GROUP_BY,
     termSize: DEFAULT_VALUES.TERM_SIZE,
     searchType: SearchType.esqlQuery,
-    sourceFields: sourceFields ?? DEFAULT_VALUES.SOURCE_FIELDS,
+    sourceFields: DEFAULT_VALUES.SOURCE_FIELDS,
```

Comment:
`DEFAULT_VALUES.SOURCE_FIELDS` seems to be `[]`, and this seems to be the only reference to it.

Reply:
Oh true, I can remove it.

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/184541/commits/e84f29566ccbd83b7d4247303006192474008274